packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.mdx

```diff
@@ -5,13 +5,20 @@ import { RadioButtonGroup } from './RadioButtonGroup';
 
 # RadioButtonGroup
 
-`RadioButtonGroup` is used to select a single value from multiple mutually exclusive options.
+`RadioButtonGroup` is used to select a single value from a set of mutually-exclusive options, arranged in a _horizontal_ control.
 
 ### When to use
 
-Use `RadioButtonGroup` for mutually exclusive selections if there are up to four options available. This is because the `RadioButtonGroup` cannot have more than one row and should still accommodate small resolutions. For a mutually exclusive selection of more than four options, use `Select` component.
+- When you have roughly three or more _mutually-exclusive_ options
```

Comment:
Three or _more_ options?

Reply:
Yeah, I think the language could be refined. Initially it said ‘up to four’, but since for just two options you should use a `Switch` by the strictest definition it should only be used when you have 3-4 options. _However_, if you’re using it with icons, this limit can be relaxed a bit (although I’m not familiar with how often this scenario comes up?). Maybe I should just be explicit about this criteria here too.